dpub/core.py

```diff
@@ -95,21 +128,27 @@ def _read_tests_map(drive,
     ids = drive.read(doc, t_range, read_dimension)
 
     tests_map = {}
+    empty_ids = 0
     for id in ids:
         # Create a test object only if the read test identifier is not empty.
         # Otherwise we are reading an empty row.
         if id and len(id) > 0:
             # Read at m_loc. If m_loc content is not empty at the sheet, skip the
             # test. Only destination empty cells will be written
             if _is_location_empty(drive, doc, m_loc):
-                tests_map[id] = Test(id, m_loc)
+                update = "Yes"
+                tests_map[id] = Test(id, m_loc, update)
```

Comment:
you can extract this sentence in common factor. It is present int the if and the else

Reply:
Yes, you're right. I'll do it.

Reply:
Done.